docs/pages/quickstart_guide.mdx

```diff
@@ -0,0 +1,78 @@
+```console
+which pelican
+```
+which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](./install.mdx) to acquire a working installation.
+
+### Useful Terminology
+**Objects:**
+Pelican should be thought of as a tool that works with federated *objects* as opposed to *files*. The reason for this is that calling something a file carries with it the connotation that the file is mutable, i.e. its contents can change without requiring a new name. Objects in a Pelican federation, however, should be treated as **immutable**, especially in any case where objects are pulled through a cache (which will be the case for almost all files in the [Open Science Data Federation](https://osg-htc.org/services/osdf), or OSDF). This is because the underlying cache mechanism, powered by [XRootD](https://xrootd.slac.stanford.edu/), will deliver whatever object it already has access to; if an object name changes at the origin, the cache will remain unaware and continue to deliver the old object. In the worst case, when the cache only has a partial object, it may attempt to combine its stale version with whatever exists at the origin. Use object names wisely!
+
+
+**Federations:**
+Objects in Pelican belong to *federations*, which are aggregations of data that are exposed to other individuals in the federation. Each Pelican federation constitutes its own global namespace of objects and each object within a federation has its own path, much like files on a computer. Fetching any object from a federation requires at minimum two pieces of information: a URL indicating the object's federation and the path to the object within that federation (there is the potential that some objects require access tokens as well, but more on that later). For example, the OSDF’s federation URL is https://osg-htc.org and an example object from the federation can be found at the following path:
+```console
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+**Namespace Prefixes:**
+Each origin supports one or more *namespace prefixes*, which are analogous to the folders or directories from your computer that you use to organize files. Below is an example object:
+```bash
+/demo/testfile.txt
+```
+This is a simple example where the namespace prefix would be `/demo` and the object is named `/testfile.txt`.
+
+However, things can get a little tricky when you add a longer namespace prefix and/or a longer filepath within that namespace where the object lives. For example:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+In this example, the namespace prefix is `/ospool/uc-shared/public/`, and the actual object is named `/OSG-Staff/validation/test.txt`. While the namespace prefix always comes before the object name, it can get a little confusing once paths and names get longer.
```

Comment:
If we say it's confusing, we should also provide information to help disambiguate the two concepts. I'm not sure the best place to point people to, but one option is to point them to their federation's registry (eg https://osdf-registry.osg-htc.org/view/registry/), which should have a list of the Pelican namespaces available in their fed. This at least gives them a way to figure out what belongs to the namespace and what belongs to the object name.

@haoming29 @CannonLock do you have a better resource to help users figure this stuff out?

Reply:
Yeah if anyone has any tips on how to figure out the separation between namespace prefix and object path for users I'd love to know so I can include it. I just wasn't sure if we had anything for them to use.

Reply:
I think the best way is to look up the registry web UI, but that would be a non-trivial task for non-OSDF users, as the only way we know for sure to get the registry URL is by looking at the federation metadata (for the client user), which is not a good user experience. 

Since the director hosts the federation discovery URL, I think it would be nice if the director web UI can have a pointer to the registry web UI. This way it's more feasible for the client user to explore the federation. I will open a new ticket for this. For now, we can say contact your federation admin for how to access the registry (which also doesn't' quite help as we don't ask for federation/director admin to provide their contact and display it in the director UI... Life is hard.)

Reply:
Or even better, why not have an endpoint at the director that will redirect user to the registry instead? Something like `https://director.org/registry`

Reply:
IMO, I think since @haoming29 made an issue to improve this, we should just maybe keep this and say it can be tricky. I can possibly add a note for now saying we are working on improving ways for the user to explore the federation and learn the differences or something. Let me know if this approach is okay with you guys. Check out the change once it is committed and see what you think.